src/helper/SM4T_translate.py

```diff
@@ -2,9 +2,317 @@
 from fastapi import HTTPException
 from transformers import AutoTokenizer, SeamlessM4Tv2ForTextToText
 
+# TODO: this should be dynamically, currently it just loads it on start
 model = SeamlessM4Tv2ForTextToText.from_pretrained("facebook/seamless-m4t-v2-large")
 tokenizer = AutoTokenizer.from_pretrained("facebook/seamless-m4t-v2-large")
 
+# From https://huggingface.co/facebook/seamless-m4t-v2-large#supported-languages
+possible_languages = [
```

Comment:
I would recommend moving this to a separate file to improve readability. Also please make this upper case because this is a const

Reply:
Done